packages/connection/src/Manager.php

```diff
@@ -1735,4 +1735,23 @@ public function sign_role( $role, $user_id = null ) {
 
 		return $role . ':' . hash_hmac( 'md5', "{$role}|{$user_id}", $token->secret );
 	}
+
+	/**
+	 * Some hosts disable the OpenSSL extension and so cannot make outgoing HTTPS requsets
+	 */
+	public static function fix_url_for_bad_hosts( $url ) {
```

Comment:
I don't see a good reason for this one to be static. As you can see in the rest of the `Manager`, there aren't many methods that are static. Could we also make this one a non-static one too?

An alternative would be to define this method in another class. Seems like it really doesn't belong anywhere though, which is why I suggested a `Utils` class.

Reply:
+1 to `Utils` class

Reply:
See a142b9c. The `fix_url_for_bad_hosts` method is now in the new Utils class.